### PR TITLE
Apply.js now gets the companies from firestore.

### LINKED ISSUE #6: Companies list displaying in main website
Depends on #5 .

In the "Get an internship" section on the website, the list that has been created should be shown as bubbles to click - just as it is know, but with the information from the list created by the administrators instead of the fixed companies that there are right know.

### PR DESCRIPTION
**Pull request by frmdz**
Resolves #6.

### CODE REVIEW

File: src/Company.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Company({ link, image, name }) {
```
Comment (dariomtz) [inline src/Company.js:4]: Lets change this so that instead of receiving the link, it receives the post_id from blogger.

**Review by dariomtz — CHANGES_REQUESTED** [review]
Looks very good to me, change so that I can approve soon

File: src/Company.js (hunk 1 above)
Comment (frmdz) [reply src/Company.js]: Done.

**Review by dariomtz — APPROVED** [review]
lgtm

**Pull request merged**